### PR TITLE
Create FahadAPoem.txt

### PR DESCRIPTION
**Pull request by fahadalgharazi**

### CODE REVIEW

File: FahadAPoem.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+poem
```
Comment (davidbtucker) [inline FahadAPoem.txt]: Hm, maybe this could have a rhyme?

File: FahadAPoem.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Roses are red
+Violets are blue 
+Approve this pull request 
+the end
```
Comment (jkschulz) [inline FahadAPoem.txt]: This is great! Can you capitalize `the` on this line for consistency please?

**Review by jkschulz — CHANGES_REQUESTED** [review]

File: FahadAPoem.txt (hunk 2 above)
Comment (fahadalgharazi) [reply FahadAPoem.txt]: Fixed it

**Review by davidbtucker — APPROVED** [review]
Haha, excellent!

**Review by davidbtucker — APPROVED** [review]

**Review by jkschulz — APPROVED** [review]

**Pull request merged**